spec/queries/keywords_query_spec.rb

```diff
@@ -4,12 +4,11 @@
 
 RSpec.describe KeywordsQuery, type: :query do
   describe '#call' do
-    context 'given user has keyword' do
+    context 'given the user searchs for ads_top_urls' do
```

Comment:
```suggestion
    context 'given the user searches for ads_top_urls' do
```

did I miss that `e` in the previous suggestion 🙈